docs/admin/admission-controllers.md

```diff
@@ -467,20 +467,22 @@ metadata:
 
 #### Internal Behavior
 This admission controller has the following behavior:
-  1. If the `Namespace` has an annotation with a key `scheduler.kubernetes.io/nodeSelector`, use its value as the
+  1. If the `Namespace` has an annotation with a key `scheduler.kubernetes.io/node-selector`, use its value as the
```

Comment:
isn't this `scheduler.alpha.kubernetes.io/node-selector` currently?

Reply:
updated accordingly.